public/js/filebrowser.js

```diff
@@ -0,0 +1,65 @@
+function itemReorder(id,pos,typ,act) { // Handles custom re-ordering of items in file browser
+    var dType = '';
```

Comment:
I'm having a hard time understanding what this function actually does... these variables could be renamed to be a bit more descriptive, but it isn't that big of a deal

Reply:
I'm going to mark this one as out-of-scope for this PR. If we want to refactor the reordering routine, I'm fine with that, but it works and I don't want to touch it. Better handled in a different PR that is purpose-built for that.

Reply:
Sounds good.